apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -222,6 +235,15 @@ export class CaseController {
       state !== CaseState.DELETED,
     )
 
+    if (
+      updatedCase &&
+      completedCaseStates.includes(updatedCase.state) &&
+      ServerSideFeatureClient.isOn('judicial-system-sqs')
```

Comment:
You should be using the `FeatureFlagService` from the `nest/feature-flags` library, it also takes in the user object to check if a specific user has access to the feature.

Reply:
I am using the `@island.is/feature-flags`. Why is that not good enough?